spec/helper/functional-test-async.helper.js

```diff
@@ -0,0 +1,211 @@
+"use strict";
+
+/**
+ * @typedef {Object} opentoken~functionalTestAsyncResponse
+ * @property {*} body Can be anything, including objects.
+ * @property {Object} links The Link header reformatted for ease of use
+ * @property {Object} headers
+ * @property {number} statusCode
+ */
+
+/**
+ * @typedef {Object} opentoken~functionalTestAsyncTest
+ * @property {Function} follow Follow a link from the last response
+ * @property {Function} request How to issue a request
+ * @property {Function} requestHandler Assigned when event is listened
+ * @property {Function} start Make a request to / and possibly follow a link
+ */
+
+var events, mockRequire, originalEventEmitter;
+
+
+/**
+ * Create an object that extends EventEmitter so EventEmitter does
+ * not have additional methods added due to Restify.
+ *
+ * Because everything uses `EventEmitter.call(this)`, this safety
+ * layer can not be a typical class.  Instead, it is wired up this way.
+ *
+ * Also, this has to be the same safety layer that is used every time,
+ * so don't go creating new safety layers continually.
+ *
+ * @return {*} Whatever EventEmitter returns
+ */
+function EventEmitterSafetyLayer() {
+    var args;
+
+    args = [].slice.call(arguments);
+
+    // eslint-disable-next-line new-cap
```

Comment:
Is this the right eslint disable line? You can't calling new here so I'm not sure what you're trying to disable.